aap_chatbot/src/AnsibleChatbot/AnsibleChatbot.tsx

```diff
@@ -303,14 +308,27 @@ export const AnsibleChatbot: React.FunctionComponent<ChatbotContext> = (
                             ref={messagesEndRef}
                           />
                         )}
-                        <div key={`m_div_${index}`}>
-                          <Message key={`m_msg_${index}`} {...message} />
-                          <ReferencedDocuments
-                            key={`m_docs_${index}`}
-                            caption="Refer to the following for more information:"
-                            referenced_documents={referenced_documents}
-                          />
-                        </div>
+                        {collapse ? (
+                          <div key={`m_div_${index}`}>
+                            <ExpandableSection toggleText="Show more">
```

Comment:
You might like to follow the example [here](https://www.patternfly.org/components/expandable-section/).

By wrapping an `ExpandableSection` in your own (for exmaple) `ExpandableSectionBasic` would allow you to toggle the text between "Show more" (when collapsed) and "Show less" (when expanded). 

At the moment your text is always "Show more" even when expanded.

Just a thought.

Reply:
Initially I thought it would be an easy change, but I saw the sample code on the PatternFly page used React' "state". Since there might be multiple ExpandableSection's, there would be multiple states and I do not want to have many states on a page. We may be able to have a different approach. I will merge this as it is and open a lower priority JIRA ticket for that.

Reply:
Ah, but the state is encapsulated in an instance of `ExpandableSectionBasic` so you can repeat as many times as you like without fear of maintaining multiple states at the _page_ level.

Reply:
You'd use your `ExpandableSectionBasic` and not PF's `ExpandableSection` directly.

Reply:
https://issues.redhat.com/browse/AAP-43218 was opened for this.